lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -88,6 +88,17 @@ public static function find(array $filters = []) {
   private function build($data) {
     $this->id = $data->sid;
     $this->created_at = $data->timestamp;
+
+    $user = user_load($data->uid);
+    $this->user = [
+      'drupal_id' => $data->uid,
+      'northstar_id' => dosomething_helpers_isset($user->uuid, 'id'),
```

Comment:
still not `uuid` @deadlybutter haha. You :heart: this whole `uuid` thing for some reason :wink: